### PR TITLE
Handle the new package structure (cont.)

### PR DESCRIPTION
**Pull request by treydock**
This is a continuation of #3

### CODE REVIEW

File: templates/redhat-access-insights.conf.erb [hunk 1]
```diff
@@ -48,8 +48,8 @@ obfuscate=<%= @obfuscate %>
 obfuscate_hostname=<%= @obfuscate_hostname %>
 <%- end -%>
 
-
-
-
-
-
+<%- if @ignore_cgroup == true -%>
```
Comment (lphiri) [inline templates/redhat-access-insights.conf.erb]: @treydock  - the "ignore_cgroup"  option has been pulled out of the insights client so it is no longer needed.

**Comment by lphiri** [comment]
@treydock @ekohl  - would like to push through and release this one - I assume we can close #3 and complete the review using this PR?

**Comment by treydock** [comment]
@lphiri That should work. This is a duplicate of #3 with some fixes and one addition.

**Review by ekohl — APPROVED** [review]
I was struggling with a proper dev/test environment for this and was very busy with other things (which is why my PR got a bit stale). The changes all look correct and I trust @treydock because he's always written high quality code.

@lphiri I'll leave it up to you merge this.

**Comment by treydock** [comment]
I've deployed this to 14 systems that are a mix of RHEL6 and RHEL7 and as far as I can tell things are working. Few days ago started getting cron emails about needing to set `ignore_cgroup=True` on my RHEL6 boxes so added that ability in this PR.

I don't think I have anything using the `access_insights_client::old` class so haven't tested that part of this.

**Review by lphiri — COMMENTED** [review]
> I've deployed this to 14 systems that are a mix of RHEL6 and RHEL7 and as far as I can tell things are working. Few days ago started getting cron emails about needing to set `ignore_cgroup=True` on my RHEL6 boxes so added that ability in this PR.
> 
> I don't think I have anything using the `access_insights_client::old` class so haven't tested that part of this.

@treydock - we are deprecating the old insights package (it does now work with new cloud services), so not critical to test it for now.   We however do need to pull out the  `cgroup` setting- like I mentioned earlier - it was introduced a few days ago but has been rolled back - its no longer needed and will become an invalid setting soon.

File: templates/redhat-access-insights.conf.erb (hunk 1 above)
Comment (treydock) [reply templates/redhat-access-insights.conf.erb]: I deleted the commit that added ignore_cgroup

**Review by lphiri — APPROVED** [review]
@treydock and @ekohl  thanks for your contributions!

**Pull request merged**